Misc/NEWS.d/next/Library/2024-12-08-08-36-18.gh-issue-127732.UEKxoa.rst

```diff
@@ -0,0 +1 @@
+Add Windows Server 2025 to ``platform.py`` detection
```

Comment:
```suggestion
The :mod:`platform` module now correctly detects Windows Server 2025.
```

We could make a true What's New entry as well. By the way, which functions are actually affected by this change? is it `platform.system()` or something like this?

Reply:
In addition, please increase the version of platform.py (now to 1.0.10). See the top-level file comment for that.

Reply:
> We could make a true What's New entry as well. By the way, which functions are actually affected by this change? is it `platform.system()` or something like this? 

Yes, this affects the `platform.win32_ver()`, and other functions that call this function are also affected, such as `platform.uname()` and `platform.system`.

Reply:
> In addition, please increase the version of platform.py (now to 1.0.10). See the top-level file comment for that.

I don't think we should change this legacy version in the docstring.

Other than the addition 3 weeks ago, the previous changes were 9, 15 and 18 years ago:

https://github.com/python/cpython/blame/70154855cf698560dd9a5e484a649839cd68dc7c/Lib/platform.py#L30

And there have been many, many updates in those 9 years:

https://github.com/python/cpython/commits/main/Lib/platform.py

The real version of the file is the Python version, and the real history is in source control and the changelog (and What's New files).

Reply:
> ```suggestion
> The :mod:`platform` module now correctly detects Windows Server 2025.
> ```
> 
> We could make a true What's New entry as well.

Suggestion applyed

Reply:
Ok for not changing the version's file (I was the one who changed it 3 weeks ago). But I still thinik a What's New entry would be good since it's a user-facing change.

Reply:
The bar for What's New isn't user-facing changes, but:

> If a change is particularly interesting for end users (for example, new features, significant improvements, or backwards-incompatible changes), add an entry in the “What’s New in Python” document (in [Doc/whatsnew/](https://github.com/python/cpython/blob/main/Doc/whatsnew/)) in addition to the NEWS entry.

https://devguide.python.org/core-developers/committing/#changes-that-require-what-s-new-in-python-entries

Do you think this is a "particularly interesting" change? I don't use Windows so it's hard for me to judge.

Reply:
Mmh. Perhaps not that "interesting" though it appeared that it previously was erroneously detected (actually, can we perhaps categorize this one as a bugfix?). If so, maybe no What's New.

Reply:
This can be a bugfix that is backported. No need for a What's new - NEWS is enough.